### PR TITLE
Add Ellipsis and Lacuna support in CMME import

### PR DESCRIPTION
**Pull request by DILewis**
Ellipsis --> gap
Lacuna --> supplied/spsace

### CODE REVIEW

**Comment by DILewis** [comment]
Added Signum Congruentiae

File: src/iocmme.cpp [hunk 1]
```diff
@@ -304,6 +306,16 @@ void CmmeInput::ReadEvents(pugi::xml_node eventsNode)
                 pugi::xml_node barlineNode = eventNode.select_node("./Barline").node();
                 CreateBarline(barlineNode);
             }
+            else if (eventNode.select_node("./Ellipsis")) {
```
Comment (lpugin) [inline src/iocmme.cpp:309]: You don't need an xpath here. Just `evenNode.child("Ellispis")` is enough, which give you the node straight way
```suggestion
            else if (eventNode.child("Ellipsis")) {
```

File: src/iocmme.cpp [hunk 2]
```diff
@@ -304,6 +306,16 @@ void CmmeInput::ReadEvents(pugi::xml_node eventsNode)
...
+                CreateEllipsis();
+            }
+            else if (eventNode.select_node("./Lacuna")) {
+                pugi::xml_node lacunaNode = eventNode.select_node("./Lacuna").node();
+                CreateLacuna(lacunaNode);
```
Comment (lpugin) [inline src/iocmme.cpp:314]:
```suggestion
                pugi::xml_node lacunaNode = eventNode.select_node("./Lacuna").node();
                CreateLacuna(eventNode.child("Lacuna"));
```

File: src/iocmme.cpp [hunk 3]
```diff
@@ -304,6 +306,16 @@ void CmmeInput::ReadEvents(pugi::xml_node eventsNode)
                 pugi::xml_node barlineNode = eventNode.select_node("./Barline").node();
                 CreateBarline(barlineNode);
             }
+            else if (eventNode.select_node("./Ellipsis")) {
+                CreateEllipsis();
+            }
+            else if (eventNode.select_node("./Lacuna")) {
```
Comment (lpugin) [inline src/iocmme.cpp:312]:
```suggestion
            else if (eventNode.child("Lacuna")) {
```

File: src/iocmme.cpp (hunk 2 above)
Comment (lpugin) [reply src/iocmme.cpp:314]: (The previous line can be removed)

File: src/iocmme.cpp [hunk 4]
```diff
@@ -304,6 +306,16 @@ void CmmeInput::ReadEvents(pugi::xml_node eventsNode)
                 pugi::xml_node barlineNode = eventNode.select_node("./Barline").node();
                 CreateBarline(barlineNode);
             }
+            else if (eventNode.child("Ellipsis")) {
+                CreateEllipsis();
+            }
+            else if (eventNode.child("Lacuna")) {
+                pugi::xml_node lacunaNode = eventNode.select_node("./Lacuna").node();
```
Comment (lpugin) [inline src/iocmme.cpp:313]:
```suggestion
```

**Pull request merged**